### DISCUSSION TITLE
Duplicate OBR and OBX segment in HL7

### DISCUSSION THREAD

**Discussion by Frazou1** [nextgenhealthcare/connect#6079] [Q&A]
Hi, i receive HL7 from a engin and sometime the message has duplicate obr and obx segment, i would like remove it before sending to destination ?

**Comment by kirbykn2** [comment]
Test for duplicate and delete.

**Reply by Frazou1** [reply]
Have you the code ?

**Reply by kirbykn2** [reply]
You can post what you have tried and what errors you are seeing, and someone will help.
